CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.14.0 Release
+- Release Date: November 16, 2018
+- Server Versions Supported: Server v4.10+ is required, Self-Signed SSL Certificates are not supported
```

Comment:
`Self-Signed SSL Certificates are not supported`

I think is misleading. They are supported if the user installs the CA cert on their device.